backend/src/routes/v0/agent/__init__.py

```
@@ -39,21 +39,21 @@
         }
     }
 )
-def list_agents(
+async def list_agents(
     user: ProtectedUser = Depends(verify_credentials), 
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)